core/tests/test_utils.py

```diff
@@ -1394,7 +1416,7 @@ def run(self, result=None):
 
     def _get_all_queue_names(self):
         """Returns a list of all queue names."""
-        return [q['name'] for q in self._testbed_taskqueue_stub.GetQueues()]
+        return [q['name'] for q in []]
```

Comment:
What's going on here? Seems like it's just returning the empty list always...

Reply:
Unneeded function, removed.